contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/VoldemortSwapJob.java

```diff
@@ -89,6 +101,89 @@ public VoldemortSwapJob(String id,
         this.failedFetchStrategyList = failedFetchStrategyList;
         this.clusterName = clusterName;
         this.buildPrimaryReplicasOnly = buildPrimaryReplicasOnly;
+        this.props = props;
+    }
+
+    private void runDistcp(Path from, Path to) throws Exception {
+        info("     source: " + from);
+        info("destination: " + to);
+        EmbeddedGobblin embeddedGobblin = new EmbeddedGobblinDistcp(from, to).mrMode();
+
+        // Used for global throttling"
+        embeddedGobblin.distributeJar("lib/*");
+
+        for (Map.Entry<String, String> entry : this.props.entrySet()) {
+            if (entry.getKey() instanceof String && ((String) entry.getKey()).startsWith("distcpConf.")) {
+                String key = ((String) entry.getKey()).substring("distcpConf.".length());
```

Comment:
Can be in either Azk job config or BnP plugin config. Ideally these attributes should be in the BnP plugin config therefore users are not aware of this change.

The assumption is that every gobblin attribute comes with a prefix (in this case "distcpConf") in order to avoid potential conflicts.This naming convention was created by the Gobblin developers. We can remove/change the "distcpConf" prefix if necessary.